src/shared/components/palette/complex/read-complex-field.component.spec.ts

```diff
@@ -12,12 +12,14 @@ import { By } from '@angular/platform-browser';
 describe('ReadComplexFieldComponent', () => {
```

Comment:
Missing test coverage - no test for new condition `if (this.caseField.display_context_parameter) {` ?

Reply:
Added